spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+RSpec.describe 'User', type: :feature do
```

Comment:
- [Required] Please consider adding a feature test that checks when a user clicks a post, it redirects the `post's` show page as requested in the project requirements. Please see the screenshot below and rectify this.

## Project requirements
![Screenshot from 2023-03-08 14-52-39](https://user-images.githubusercontent.com/16392046/223709894-2e888a1a-d7e7-4597-aac3-9bd87bc60362.png)